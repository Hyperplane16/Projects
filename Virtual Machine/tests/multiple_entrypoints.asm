FUNC LABEL 0
    CAL VAL 1
    PRINT REG 0
    RET
FUNC LABEL 1
    PRINT VAL 4
    RET
FUNC LABEL 0
    RET
