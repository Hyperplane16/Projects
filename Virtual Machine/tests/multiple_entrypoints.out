2 functions of label 0 found.
1